### ISSUE TITLE
Inserting classes of unknown type into the schema graph is not checked properly

### ISSUE THREAD

**Issue by matko** [terminusdb/terminusdb#437]
It is possible to insert a type definition into a schema like the following:
```
{"@id":"Moooo",
 "@type":"This_Type_Has_No_Meaning",
 "name":"xsd:string"}
```
Even though the given type is unknown in our schema language this will still be accepted. For the above example, it will insert a type triple, as well as a triple for the `name` property definition, so it appears this is treated like a Class. Subsequent document queries by id can retrieve this definition, but retrieving the full schema will skip over it, since this only retrieves classes of known types.

We should make sure that everything in the schema has a type known to the checker, so we don't end up with garbage in our schemas. This is especially important to catch typos like 'Claass' instead of 'Class'.

**Comment by matko** [comment]
Omitting the type is also apparently allowed.

**Comment by matko** [comment]
This has been resolved in pull request #564.
